### PR TITLE
Fix and update rippled.spec rpm file

### PR DESCRIPTION
**Pull request by wilsonianb**
See comments in rippled.spec for changes to updating rippled version.

### CODE REVIEW

File: Builds/rpm/rippled.spec [hunk 1]
```diff
@@ -20,13 +24,11 @@ Rippled is the server component of the Ripple network.
 
 
 %prep
-%setup -n rippled-release
+%setup -n rippled-%{rippled_branch}
 
 
 %build
-# Assume boost is manually installed
-export RIPPLED_BOOST_HOME=/usr/local/boost_1_55_0
```
Comment (stevenzeiler) [inline Builds/rpm/rippled.spec]: How is the boost dependency managed now?

Comment (wilsonianb) [inline Builds/rpm/rippled.spec]: Boost is installed and linked in such a way that `RIPPLED_BOOST_HOME` isn't needed.
https://github.com/wilsonianb/rippled/blob/build-rpm/Builds/rpm/Dockerfile#L9-L14

Comment (stevenzeiler) [inline Builds/rpm/rippled.spec]: Oh cool so the rpm build must take place in the Docker container, otherwise dependencies must be installed manually.

**Comment by vinniefalco** [comment]
:+1:

**Pull request closed without merging**